<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainTab">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QFrame" name="frameKeys">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="margin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QTableWidget" name="keyTable">
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
               <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Label</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Key</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Mode</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Description</string>
                </property>
               </column>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="btnAddKey">
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnModKey">
                  <property name="text">
                   <string>Modify</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnDelKey">
                  <property name="text">
                   <string>Delete</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QFrame" name="frameVideo">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QGraphicsView" name="videoView">
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="backgroundBrush">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </property>
               <property name="foregroundBrush">
                <brush brushstyle="NoBrush">
                 <color alpha="255">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </property>
               <property name="cacheMode">
                <set>QGraphicsView::CacheNone</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="labelVideoStatus">
                  <property name="text">
                   <string>-</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnPlay">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>|&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnPause">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>| |</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btnRec">
                  <property name="maximumSize">
                   <size>
                    <width>31</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>O</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QFrame" name="framePlot">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="GraphicsLayoutWidget" name="graphicsLayoutView"/>
            </item>
            <item>
             <widget class="QLabel" name="labelPlotStatus">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="configTab">
       <attribute name="title">
        <string>Configuration</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="ParameterTree" name="parameterView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>ParameterTree</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph.parametertree</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
